ftva_lab_data/tests.py

```
@@ -239,6 +239,26 @@ def test_search_results_with_pagination(self):
         self.assertIn("test_file_10", page_2_content)
         self.assertNotIn("unique_file_1", page_2_content)
 
+    def test_items_per_page_control(self):
+        url = reverse("render_table")
+        test_values = ["", 10, 20, 50, 100, "foobar"]
```

The 2 string values could use some clarification.  From reading the code in the view, I understand that if the value can't be cast to `int`, the default value will be used and the exception swallowed.  But there's nothing in this test related to that, so the test is not intuitive.

In fact, I think it would be better to add a test explicitly for non-int values - one should be enough, since `int("")` and `int("foobar")` both give a `ValueError`.  Then the test can assert that `pagination_per_page` is the default value.... which might lead to a little refactor to move the options into a dedicated function, so they're easier to include in tests or other reuse.

---

I added a new test to address invalid options. I see what you mean about only needing one string value, so I kept just the empty string. Then it occurred to me that a Boolean might cause an issue, so I added a Boolean as an invalid test value. Still works, as `int()` still throws a `ValueError`.